src/main/java/ui/Ui.java

```diff
@@ -317,11 +322,24 @@ public void printNoteDeleteInstructions() {
     }
 
     public void printDeleteAllNoteMessage() {
-        System.out.println(DELETE_NOTE_MESSAGE);
+        System.out.println(DELETE_ALL_NOTE);
+        System.out.println();
     }
 
     public void printNoteMissingError(int size) {
         System.out.println("Invalid index! There are only " + size + " notes currently.");
+        System.out.println();
```

Comment:
Maybe can add \n behind the first line instead of putting an empty System.out.println()?